DisplaySettings/DisplaySettings.json

```diff
@@ -951,6 +951,17 @@
         },
         "getSinkAtmosCapability":{
             "summary": "Returns the ATMOS capability of the sink (HDMI0).",
+            "params": {
+                "type":"object",
+                "properties": {
+                    "audioPort": {
+                        "$ref": "#/definitions/audioPort1"
```

Comment:
could you fix why we have two definition for audioPort1 audioPort0?

Reply:
Not sure, this is not part of my change.